nvm.sh

```diff
@@ -2665,6 +2671,7 @@ nvm() {
         nvm_echo '  nvm install [<version>]                     Download and install a <version>. Uses .nvmrc if available and version is omitted.'
         nvm_echo '   The following optional arguments, if provided, must appear directly after `nvm install`:'
         nvm_echo '    -s                                        Skip binary download, install from source only.'
+        nvm_echo '    -b                                        Skip source download, install from binary only.'
```

Comment:
we should ensure that `-s -b` errors.

Reply:
Yeah so with respect to this if the user specifies `-s` line number https://github.com/nvm-sh/nvm/blob/master/nvm.sh#L3167 will not execute anyways so I was thinking maybe pass the {nosource} parameter to `nvm_install_source` and then check if its set inside we throw an error ? Does this approach sound okay ?

Reply:
I think it's better to check for conflicting arguments at argument parse time, so that those errors come as soon as possible - with your suggestion, if there's no binary available on nodejs.org, it wouldn't error on `-s -b`.

Reply:
Yeah made the change